test/libsolidity/util/TestFileParser.cpp

```diff
@@ -282,8 +282,8 @@ Parameter TestFileParser::parseParameter()
 			if (parameter.alignment != Parameter::Alignment::None)
 				throw Error(Error::Type::ParserError, "String literals cannot be aligned or padded.");
 
-			parameter.abiType = ABIType{ABIType::String, ABIType::AlignLeft, 32};
 			string parsed = parseString();
+			parameter.abiType = {ABIType::String, ABIType::AlignLeft, parsed.size()};
```

Comment:
Is it later padded to multiples of 32 bytes?

Reply:
Yes, that's done later in `BytesUtils::applyAlign()`.